### PR TITLE
Favicons

### PR DESCRIPTION
**Pull request by gentlecat**
:sparkles: :volcano: :sparkles:

### CODE REVIEW

**Comment by mayhem** [comment]
Yay for favicons! \ø/ However, I think we should not be using the full size logo for the the larger favicons. Let's use the medium sized ones that have a little bit less detail in them.

**Comment by gentlecat** [comment]
Oh, right. Picked wrong one for some reason.

**Comment by gentlecat** [comment]
@mayhem, looks better?

File: templates/base.html [hunk 1]
```diff
@@ -8,7 +8,12 @@
     <title>{% block title %}AcousticBrainz{% endblock %}</title>
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="icon" href="{{ url_for('static', filename='img/favicon.png') }}" type="image/x-icon" />
+
+    <link rel="icon" sizes="16x16" href="{{ url_for('static', filename='img/favicon-16.png') }}" type="image/png" />
```
Comment (alastair) [inline templates/base.html]: do we really need so many favicons? what are all of these used for? homscreen icons on an ipad? all my browser windows have 16x16 icons :)

Comment (mayhem) [inline templates/base.html]: You actually use devices that uses favicons? Wow, so modern. :)

more seriously, the list of actual favicons that should be supported to make everything in the world happy is totally redonkulous. I think this is a fair balance.

Comment (gentlecat) [inline templates/base.html]: They are used on high-res screens, [homescreen stuff](https://developer.chrome.com/multidevice/android/installtohomescreen#supporting) on mobile devices, etc.

**Comment by gentlecat** [comment]
Is this OK to merge?

**Comment by mayhem** [comment]
Sorry, didn't see the update. Yep! :+1:

**Pull request merged**